### DISCUSSION TITLE
VL53L1x Update to current Arduino ver 2.1.0

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#15028] [Q&A] [answered]
Tasmota is using version 1.02 of this Arduino driver, the current version is 2.1.0.
I've noticed issues with the current version, returning distant values of 9999mm for extended periods of time.
Reboot was needed to clear the error.

Please consider this update...  

Thanks
~~~
https://www.arduino.cc/reference/en/libraries/stm32duino-vl53l1/
~~~

**Comment by Jason2866** [comment, marked as answer]
Tasmota is not using this lib. It uses https://github.com/pololu/vl53l1x-arduino
There is no mandantory change in the lib.
